docs/source/notes/hateful_memes_challenge.md

```diff
@@ -0,0 +1,61 @@
+# Hateful Memes challenge
+
+Hateful Memes challenge is available at [this link](https://www.drivendata.org/competitions/64/hateful-memes).
+
+We provide the baseline pretrained models for this challenge and the configurations used for training the reported baselines. For details check [here](https://github.com/facebookresearch/mmf/tree/master/projects/hateful_memes).
+
+Here we provide a step-by-step tutorial on how to setup your mmf codebase for development, running training and evaluation of your models on hateful memes dataset and generating submission file for the challenge.
+
+## Installation
```

Comment:
I would not replicate the content here. Instead, it would be better to only talk about challenge prediction specification. For others point to the README.

Reply:
But shouldn't tutorial have an example(here using mmbt) showing how to do it? In addition to that, downloading data is similar to Readme i agree but it is also essential for the tutorial flow.

Reply:
Updated by removing redundant parts.